portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,11 +27,49 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    // Create an arraylist object with hard-coded messages
+    private ArrayList<String> messages;
 
+  @Override
+  public void init() {
+    messages = new ArrayList<String>();
+    messages.add("Good Morning!");
+    messages.add("Good Afternoon!");
+    messages.add("Good Evening!");
+    messages.add("Good Night!");
+  }
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
+    // Convert the ArrayList to JSON
+    Gson gson = new Gson();
+    String json = gson.toJson(messages);  
+    
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+    // response.setContentType("text/html;");
     //why doesnt the headers in html format work and only show up as chars?
-    response.getWriter().println("Hello Mariah! How are you today?");
+    //response.getWriter().println("Hello Mariah! How are you today?");
+  }
+
+  /**
+   * Converts a ServerStats instance into a JSON string using manual String concatentation.
+   */
+  private String convertToJson(ArrayList<String> messages) {
+    String json = "{";
+    json += "\"morning\": ";
+    json += "\"" + messages.get(0) + "\"";
+    json += ", ";
+    json += "\"afternoon\": ";
+    json += "\"" +  messages.get(1) + "\"";
+    json += ", ";
+    json += "\"evening\": ";
+    json += "\"" + messages.get(2) + "\"";
+    json += ", ";
+    json += "\"night\": ";
+    json += "\"" + messages.get(3) + "\"";
+    json += "}";
+    return json;
```

Comment:
Let's remove this function now that it is no longer used.

Reply:
Done